docs/platform_concepts/inbox/chats-configuration.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Chats Configuration
+sidebar_label : Chats Configuration
+---
+
+
+Chat configurations define the functionality that is available to your agents and support them. Create configurations to give different capabilities to multiple types of users. Chat configurations enable you to control your users’ access to certain Chat features. You can create multiple configurations and assign them to different types of users.
+
+
+1. From [yellow.ai](https://cloud.yellow.ai/bot/Bot_ID/overview), select Inbox
+
+![](https://i.imgur.com/tGxzwi4.png)
+
+2. Click on Settings
+
+![](https://i.imgur.com/hec2r3b.png)
+
+3. From the drop down, select Chats configuration
+
+
+### Tags
+
+
+
+
+
+
+### Chats timeout
+
+
+
+
+
+### Custom Fields
+
+
+The custom fields setting is used to define and add fields that are required to create, transfer or close tickets.
+
+
+![](https://i.imgur.com/TVzPBUv.png)
+
+
```

Comment:
Move this content to custom-fields file located at: `docs/platform_concepts/inbox/inbox-settings/chats-configuration/custom-fields.md`

Reply:
There is already a Custom field doc - https://docs.yellow.ai/docs/platform_concepts/inbox/Chat%20custom%20fields
& 
https://docs.yellow.ai/docs/platform_concepts/inbox/inbox-settings/chats-configuration/custom-fields

Did not want to replicate.
This has to be checked first.